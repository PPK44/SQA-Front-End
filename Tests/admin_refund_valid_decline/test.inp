LOGIN
ADMINBOY
AdPass123
REFUND
test123
test
90
logout
exit