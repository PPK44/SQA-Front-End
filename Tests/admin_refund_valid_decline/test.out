
==========================================
      SQA Baller Squad Auction House
==========================================

Please Type login, create or exit!
Enter Command: 
=========================================================
                        LOGIN
=========================================================

Username: 
Password: 
Welcome ADMINBOY! Glad to see you are back :)
Your user Type is AA.

Enter Command: 
=========================================================
                        REFUND
=========================================================

Please enter seller's name: 
Please enter buyer's name: 
Please enter credit amount to refund: 
Money has been refunded, moving you back to Main Menu!
Enter Command: 
==========================================
      SQA Baller Squad Auction House
==========================================

Please Type login, create or exit!
Enter Command: 

